extensions/mssql/src/objectManagement/ui/findObjectDialog.ts

```diff
@@ -107,14 +117,16 @@ export class FindObjectDialog extends DialogBase<FindObjectDialogResult> {
 		this.objectsLoadingComponent.loading = true;
 		this.findButton.enabled = false;
 		try {
-			const results = await this.objectManagementService.search(this.options.contextId, <mssql.ObjectManagement.NodeType[]>this.selectedObjectTypes);
+			const results = await this.objectManagementService.search(this.options.contextId, this.selectedObjectTypes);
```

Comment:
`this.selectedObjectTypes.map(item => this.getObjectTypeName(item))`?

Reply:
the selectedObjectTypes are meant to store names, not display names, let me have a look.

Reply:
issue fixed with latest commit.